### PR TITLE
Support Locust 1.0.3

### PR DESCRIPTION
**Pull request by youngkih**
This is to support Locust 1.0.3, which includes updates like option names, etc 

### Related Issue
 https://github.com/karol-brejna-i/docker-locust/issues/15

### CODE REVIEW

File: examples/1.0.3/docker-compose-distributed-headless.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      QUIET_MODE: "${QUIET_MODE:-false}"
+      LOCUST_OPTS: "--headless --expect-workers 1 -u 1 -r 1 -t 10s"
+    volumes:
+      - ./locust-scripts:/locust
+
+  slave:
+    image: "grubykarol/locust:1.0.3-python3.8-alpine3.11"
+    environment:
+      ATTACKED_HOST: http://example.com
+      LOCUST_MODE: worker
```
Comment (youngkih) [inline examples/1.0.3/docker-compose-distributed-headless.yml:22]: from Locust 1.0.x, `slave` is replaced with `worker`, so LOCUST_MODE had to be updated accordingly

File: examples/1.0.3/docker-compose-distributed.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      TASK_DELAY_TO: 30
+      QUIET_MODE: "${QUIET_MODE:-false}"
+    volumes:
+      - ./locust-scripts:/locust
+
+  slave:
+    image: "grubykarol/locust:1.0.3-python3.8-alpine3.11"
+    environment:
+      ATTACKED_HOST: http://master:8089
+      LOCUST_MODE: worker
```
Comment (youngkih) [inline examples/1.0.3/docker-compose-distributed.yml:25]: Same as above

File: examples/1.0.3/locust-scripts/locustfile.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+class TestBehaviour(TaskSet):
+    @task
+    def task1(self):
+        log("running task1")
+        self.client.get("/")
+
+
+class TestUser(HttpUser):
+    tasks = [TestBehaviour]
```
Comment (youngkih) [inline examples/1.0.3/locust-scripts/locustfile.py:26]: Same as above, Locust 1.0.x is now recommending to use 
- `HttpLocust` -> `HttpUser`
- `task_set` -> `tasks`

**Comment by karol-brejna-i** [comment]
@youngkih thank you for this PR. give me a couple of days to review.

**Comment by youngkih** [comment]
@karol-brejna-i Any updates on this?

**Comment by karol-brejna-i** [comment]
I am back from vacation. Starting testing today.

**Comment by karol-brejna-i** [comment]
@youngkih Thank you for this contribution. And thank you for thinking about a dedicated example for 1.0.x versions

**Pull request merged**